origins_ckeditor_enhancements/origins_ckeditor_enhancements.module

```diff
@@ -31,3 +31,15 @@ function remove_format_help_process_text_format_element(&$element, FormStateInte
   }
   return $element;
 }
+
+/**
+ * Implements hook_editor_js_settings_alter().
+ */
+function origins_common_editor_js_settings_alter(array &$settings) {
```

Comment:
This looks like the wrong function signature. The module is `origins_ckeditor_enhancements` but the hook prefix is `origins_common`

Given the specific nature of the change and module, I'd suggest:

```
function origins_ckeditor_enhancements_editor_js_settings_alter(array &$settings) {
```

Reply:
Good spot - I originally had the function in origins common before spotting the origins ckeditor enhancements module. DOH!